types/test/unzipObjWith.ts

```diff
@@ -0,0 +1,3 @@
+import * as RA from 'ramda-adjunct';
+
+RA.unzipObjWith((v, t) => [t, 1], { a: 'b' }); // $ExpectType [string[], number[]]
```

Comment:
Let's add more tests + we're missing tests for currying

Reply:
I don't have a huge amount of experience with typescript and type tests- I've added some more based on what I can see / understand of the other tests in the folder. But I'll need some suggestions of things to test if these don't meet what you're expecting.